gulpfile.js

```diff
@@ -4,17 +4,19 @@ const gulp = require('gulp');
 const jasmine = require('gulp-jasmine');
 const istanbul = require('gulp-istanbul');
 const SpecReporter = require('jasmine-spec-reporter');
+const coveralls = require('gulp-coveralls');
+const IS_CI = process.env.CI || false;
```

Comment:
I think you should think about the problem should be: if the script is running on a remote server. So I'm suggest you are using `NODE_ENV=local` to split the branches, thank you.